app/models/concerns/searchable_model.rb

```diff
@@ -68,14 +68,25 @@ module SearchableModel
         end
       else
         unless attrs.empty?
+          # quick fix to enable searching by repositoy_row id
```

Comment:
Me no like this fix :banana:

Reply:
the whole model is a big :banana: that needs to be refactored and tested someday